docs/samples/browser-plugin-meetings/app.js

```diff
@@ -427,19 +427,26 @@ function refreshCaptcha() {
 meetingsListElm.onclick = (e) => {
   selectedMeetingId = e.target.value;
   const meeting = webex.meetings.getAllMeetings()[selectedMeetingId];
+  const selectedMeetingType = createMeetingSelectElm.options[createMeetingSelectElm.selectedIndex].innerText;
 
   if (meeting && meeting.passwordStatus === 'REQUIRED') {
     meetingsJoinPinElm.disabled = false;
     verifyPasswordElm.disabled = false;
     document.getElementById('btn-join').disabled = true;
     document.getElementById('btn-join-media').disabled = true;
   }
-  else if (meeting && meeting.passwordStatus === 'UNKNOWN') {
+  else if (meeting && (meeting.passwordStatus === 'UNKNOWN' && selectedMeetingType==="SIP URI" )) {
     meetingsJoinPinElm.disabled = true;
     verifyPasswordElm.disabled = true;
     document.getElementById('btn-join').disabled = true;
     document.getElementById('btn-join-media').disabled = true;
   }
+  else if(meeting && (meeting.passwordStatus === 'UNKNOWN' && selectedMeetingType!="SIP URI")) {
```

Comment:
Does this mean, anything other than SIP URI would go through this flow? Which is one of the following,
- Email
- Person ID
- Conversation URL
- One to One Meeting

Reply:
Also, please provide spaces between in-fix operators always and follow single quote,
```suggestion
  else if(meeting && (meeting.passwordStatus === 'UNKNOWN' && selectedMeetingType != 'SIP URI')) {
```

Reply:
1) Hi @mkesavan13,when we given Email or Person ID or One to One Meeting thats create an locus call and for Conversation URL and SIP URI the Password Status was different from UnKnown.Why I keep SIP URI because for the Schedule Meeting we need a Password and for Other options we dont need password as it create an instant locus call.
2)I will Update the Code.